addons/danger/functions/fnc_shareInformationRange.sqf

```diff
@@ -30,27 +30,22 @@ _range = switch (side _unit) do {
 _range = _range min viewDistance;
 
 // Sort long range radios
-_radio = false;
-{
-    // in vehicle (and close), has backpack, or TFAR radio
-    // or lambs_danger_fnc_dangerRadio is set to true
-    _hasRadio = (
-        _x getVariable [QGVAR(dangerRadio),false]
+private _target = objNull;
```

Comment:
Am I misreading something here? If the function finds no unit with a backpack, it still needs to return the calling unit. That unit will be the origin of the radio transmission.

From a design point of view: The backpack/variable function exists just on the border of the scope. Adding a backpack is a mission makers option, but not integral to the functioning of the mod. Hence the generous default communications range.

The reason this function needs to return Unit, range and radio, is to allow for scripts that will only trigger when a long range radio is present. (Which might be useful in an artillery/support rewrite).

Reply:
aha now it makes more sense what that function does do. will be fixed in next push

Reply:
#needs_better_code_headers